securedrop/client.py

```diff
@@ -37,31 +41,86 @@ def make_dict(self):
         }
 
 
+class AESWrapper(object):
+
+    def __init__(self, key):
+        self.bs = AES.block_size
+        self.key = hashlib.sha256(key.encode()).digest()
```

Comment:
The encryption key shouldn't be the key hash. If that is the case, if the DB is compromised, the attacker can easily decrypt, since the key hash is stored.

Reply:
Wow yea..  idk what I was thinking.